packages/core/src/components/portal/portal.md

```diff
@@ -40,15 +40,16 @@ components which use portals (popovers, tooltips, dialogs, etc.). You can do so
 ```tsx
 import { Button, Popover, PortalProvider } from "@blueprintjs/core";
 import * as React from "react";
-import * as ReactDOM from "react-dom";
+import * as ReactDOM from "react-dom/client";
 
-ReactDOM.render(
+const root = ReactDOM.createRoot(document.getElementById("root"));
```

Comment:
Opting to use language most similar to what is used in the [React docs](https://react.dev/reference/react-dom/client/createRoot#reference).